buildSrc/src/main/kotlin/Projects.kt

```diff
@@ -1,11 +1,24 @@
-import org.gradle.kotlin.dsl.DependencyHandlerScope
-import org.gradle.kotlin.dsl.project
+import org.gradle.api.Project
+import java.io.ByteArrayOutputStream
 
 /**
  * whether the process has been invoked by JitPack
  */
 val isJitPack get() = "true" == System.getenv("JITPACK")
 
+val Project.commitHash: String
```

Comment:
Github allows you to get the commit hash, why is this logic code introduced.

Reply:
You are referring to the `GITHUB_SHA` environment variable, right? This one is the full hash, @DRSchlaubi wanted to have the short one if available (retrieved by `git rev-parse --short HEAD`).

Reply:
Github doesn't provide that? in the environment?

Reply:
I didn't search too long, maybe they do. Do you know more?

Reply:
> Github doesn't provide that? in the environment?

https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables only lists the long SHA

Reply:
Maybe we should compute the short hash in the actions instead (like [here](https://github.community/t/github-context-with-short-sha/17426)).
It would probably also be good to provide `KORD_COMMIT_HASH` and `KORD_SHORT_COMMIT_HASH`.